storage-access.bs

```diff
@@ -155,25 +158,34 @@ When invoked on {{Document}} |doc|, the <dfn export method for=Document><code>ha
 1. If |global| is not a [=secure context=], then [=/resolve=] |p| with false and return |p|.
 1. If |doc|'s [=Document/browsing context=] is a [=top-level browsing context=], [=/resolve=] |p| with true and return |p|.
 1. If the [=top-level origin=] of |doc|'s [=relevant settings object=] is an [=opaque origin=], [=/resolve=] |p| with false and return |p|.
-1. If |doc|'s [=Document/origin=] is [=same origin=] with the [=top-level origin=] of |doc|'s [=relevant settings object=], [=/resolve=] |p| with true and return |p|.
+1. If |doc|'s [=Document/browsing context=] is same authority with |doc|'s [=Document/browsing context=]'s [=top-level browsing context=], [=/resolve=] |p| with true and return |p|.
+
+ISSUE: "same authority" here is a placeholder for a future concept that allows user agents to perform [=same site=] checks while adhering to additional security aspects such as the presence of a cross-site parent document, see [whatwg/storage#142](https://github.com/whatwg/storage/issues/142#issuecomment-1122147159). For the purpose of implementation, the user agent should use this step to determine whether the embedded document needs to invoke {{Document/requestStorageAccess()}} to be granted storage access. This might involve comparing the [=site for cookies=] or performing a [=same site=] check with the top-level document.
```

Comment:
```suggestion
  ISSUE: "same authority" here is a placeholder for a future concept that allows user agents to perform [=same site=] checks while adhering to additional security aspects such as the presence of a cross-site parent document, see [whatwg/storage#142](https://github.com/whatwg/storage/issues/142#issuecomment-1122147159). In practice, this might involve comparing the [=site for cookies=] or performing a [=same site=] check with the top-level document.
```
Would this work here? We didn't really describe the purpose of this step before. Not sure why we'd start now.

Reply:
Yup, that seems good! Thanks for adding the suggestion :)